karibu-dsl-v23/src/main/kotlin/com/github/mvysny/karibudsl/v23/Dialogs.kt

```diff
@@ -97,6 +116,23 @@ public fun ConfirmDialog.setConfirmIsDanger() {
     setConfirmButtonVariant(ButtonVariant.LUMO_ERROR, ButtonVariant.LUMO_PRIMARY)
 }
 
+/**
+ * Sets the confirm button as dangerous, e.g. when you're confirming a deletion action
+ * that can not be reversed.
+ *
+ * Example of usage:
+ * ```kotlin
+ * setConfirmButton(buildSingleComponent { button("Delete", VaadinIcon.CHECK.create()) {
+ *          setConfirmIsDanger()
+ *          onClick {...}
+ *      }
+ * })
+ * ```
+ */
+public fun Button.setConfirmIsDanger() {
```

Comment:
Please add this to karibu-tools instead. Also, let's name it `Button.setDanger()`.

Reply:
See https://github.com/mvysny/karibu-tools/pull/11